server/py_files/models/CNNClassifier/CNNClassifier.py

```diff
@@ -37,17 +37,17 @@ def train(self, samples, labels):
         else:
             raise Exception('Please select a valid feature')
 
-        model.add(Conv1D(256, 5, activation='relu', data_format='channels_first', padding='valid', strides=4))
-        model.add(MaxPool1D(5))
-        model.add(Dropout(0.3))
-        model.add(Conv1D(128, 5, activation='relu', data_format='channels_first', padding='valid', strides=2))
-        model.add(MaxPool1D(50))
+        model.add(Conv1D(filters=256, kernel_size=5, strides=4, padding='valid', activation='relu', data_format='channels_first'))
+        model.add(MaxPool1D(pool_size=5))
+        model.add(Dropout(rate=0.3))
+        model.add(Conv1D(filters=128, kernel_size=5, strides=2, padding='valid', activation='relu', data_format='channels_first'))
+        model.add(MaxPool1D(pool_size=50))
         model.add(Flatten())
-        model.add(Dense(64, activation='relu'))
+        model.add(Dense(units=64, activation='relu'))
         model.add(Dropout(0.5))
-        model.add(Dense(32, activation='relu'))
-        model.add(Dropout(0.5))
-        model.add(Dense(5, activation='softmax'))
+        model.add(Dense(units=32, activation='relu'))
+        model.add(Dropout(rate=0.5))
+        model.add(Dense(units=5, activation='softmax'))
```

Comment:
@astrebeck : noting a point for discussion => not sure how this is working even `model.add(Dense(units=5, activation='softmax'))` on your end => this should throw a data shape mismatch error as our `num_classes = 4`

On the same note, updating this to `model.add(Dense(units=self.num_classes, activation='softmax'))` in the next commit => this way we never end up with such situations